functions.php

```diff
@@ -34,34 +38,71 @@ public function __construct() {
 		add_action( 'init', array( $this, 'register_taxonomies' ) );
 		add_action( 'cmb2_admin_init', array( $this, 'register_header_metabox' ) );
 		add_action( 'pre_get_posts', array( $this, 'tags_support_query' ) );
+		add_action( 'admin_init', array( $this, 'add_copyright_text' ) );
 
 		// Default actions are in: https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-includes/default-filters.php#L0
 		remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
 		remove_action( 'wp_head', 'wp_generator' );
 		remove_action( 'wp_print_styles', 'print_emoji_styles' );
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles') );
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles' ) );
 		parent::__construct();
 	}
 
+	/**
+	 * Show copyright text field.
+	 *
+	 * @param array $args
+	 */
+	public function cp_show_settings( $args ) {
+		$data = esc_attr( get_option( 'copyright', '' ) );
+
+		printf(
+			'<input type="text" name="copyright" class="regular-text" value="%1$s" id="%2$s" />',
+			$data,
+			$args['label_for']
+		);
+	}
+
+	public function add_copyright_text() {
+		add_settings_section(
+			'cp_id',
```

Comment:
Might be better to use more verbose var names to remove any guessing what that is.

Reply:
Yes I made changes